src/lib/viewers/BaseViewer.js

```diff
@@ -1256,6 +1264,18 @@ class BaseViewer extends EventEmitter {
 
         return addedElement;
     }
+
+    /**
+     * Method to exit annotation mode
+     *
+     * @return {void}
+     */
+    exitAnnotationMode = () => {
+        if (this.areAnnotationsEnabled() && this.annotator && this.annotationControls) {
+            this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
+            this.annotationControls.resetControls();
```

Comment:
I'm not sure I understand the logical flow. Can we just call `this.annotatonControls.focus()` as the last line here?